src/m365/pa/commands/app/app-get.ts

```diff
@@ -44,7 +46,9 @@ class PaAppGetCommand extends PowerAppsCommand {
     this.telemetry.push((args: CommandArgs) => {
       Object.assign(this.telemetryProperties, {
         name: typeof args.options.name !== 'undefined',
-        displayName: typeof args.options.displayName !== 'undefined'
+        displayName: typeof args.options.displayName !== 'undefined',
+        asAdmin: args.options.asAdmin === true,
```

Comment:
While this works, we usually use `!!option` so let's stick with that for consistency.